Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1280,7 +1280,7 @@ void RenderFlexibleBox::layoutFlexItems(bool relayoutChildren)
 {
     if (LayoutIntegration::canUseForFlexLayout(*this))
         return layoutUsingFlexFormattingContext();
-    Vector<LineContext> lineContexts;
+    Vector<LineState> lineStates;
```

Comment:
is it lineStates or linesState?

Reply:
Hmm yeah it seems like `linesState` might be slightly less misleading. When I read `linesState` by itself I think that there is singular state for all of the lines, but the type of the variable does give you more context to realize it holds a state object for each of the lines.